ci/travis_script_manylinux.sh

```diff
@@ -53,7 +53,7 @@ for PYTHON_TUPLE in ${PYTHON_VERSIONS}; do
     -e UNICODE_WIDTH=$UNICODE_WIDTH \
     -v $PWD:/io \
     -v $PWD/../../:/arrow \
-    quay.io/xhochy/arrow_manylinux1_x86_64_base:latest \
+    quay.io/pravindra/arrow_manylinux1_x86_64_base:latest \
```

Comment:
@xhochy unfortunately, there was a typo in the change I gave you for the boost change in build_boost.sh (I had put multi_precision instead of multiprecision).

I fixed that, and added a trigger in quay.io to build. Please let me know if that's okay. I've included the change to build_boost.sh (fixing the typo) as part of this PR.

Reply:
👍